monitor_test.go

```diff
@@ -46,3 +49,52 @@ var _ = Describe("Monitor command", Label("monitor"), func() {
 		Expect(lst[3]).To(ContainSubstring(`"set" "bap" "8"`))
 	})
 })
+
+func TestMonitorCommand(t *testing.T) {
```

Comment:
I noticed that this new test is written with the `testing` framework. I guess this is intentional, right? Just some lines above there is a similar test written with Ginkgo.

Reply:
Correct, this is just to make our debugging easier in the future